pkg/plugin/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -70,8 +70,11 @@ endif
 all: manager
 
 # Run tests
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: generate fmt vet manifests
-	go test -race -coverprofile cover.out ./...
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
```

Comment:
This link should point to the controller-runtime version used by the scaffolded project, so can you add a TODO to version this script on the next controller-runtime release?